lib/recipe/models/recipe_ingredient.dart

```diff
@@ -27,6 +27,35 @@ class Ingredient extends Equatable {
   final bool saved;
   final String id;
 
+  Ingredient.fromJson(Map<String, dynamic> json)
+      : name = Name.fromJson(json['name']),
+        description = Description.fromJson(json['description']),
+        cookingState = CookingState.fromJson(json['cookingState']),
+        measurementMethod =
+            MeasurementMethod.fromJson(json['measurementMethod']),
+        measurement = Measurement.fromJson(json['measurement']),
+        measurementUnit = MeasurementUnit.fromJson(json['measurementUnit']),
+        size = Size.fromJson(json['size']),
+        sizeNumber = SizeNumber.fromJson(json['sizeNumber']),
+        saved = json['saved'] == 'true' ? true : false,
+        id = json['id'];
+
+  Map<String, dynamic> toJson() {
+    final Map<String, dynamic> data = <String, dynamic>{};
+
+    data['name'] = name.toJson();
+    data['description'] = description.toJson();
+    data['cookingState'] = cookingState.toJson();
+    data['measurementMethod'] = measurementMethod.toJson();
+    data['measurement'] = measurement.toJson();
+    data['measurementUnit'] = measurementUnit.toJson();
+    data['size'] = size.toJson();
+    data['sizeNumber'] = sizeNumber.toJson();
+    data['saved'] = saved.toString();
+    data['id'] = id;
+    return data;
```

Comment:
Great! I haven't tried this out but looks that it should be good for local storage via shared preferences 😊.